Makefile

```diff
@@ -14,6 +14,9 @@ lint: ## Run the ruff python linter (auto triggered on pre-commit)
 test: ## Run pytest and check test coverage (auto triggered on pre-push)
 	pipenv run pytest --cov-report term-missing --cov=src --cov-config=./tests/coverage.rc
 
+validate: ## Validate stub data files
+	pipenv run python3 ./src/store/metadata/stub/validation.py
+
 populate: ## Populate an oxigraph DB (that/if/when) one is running on localhost:7878
 	pipenv run python3 ./src/data.py
```

Comment:
can we add the validate make command to run prior to this one (chaining make commands is a thing you can do). Lets 100% make sure nobody skips a step and tries to populate the graph with incorrect data.

Reply:
I think this works...